owl_msgs/README.md

```diff
@@ -0,0 +1,21 @@
+The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for
+Exploration Research and Simulation can be found in README.md in the
+root directory of this repository.
+
+owl_msgs
+===================
+
+This package contains OceanWATERS and OWLAT telemetry. The messages
+defined here are currently used for fault detection. The
+detection is relayed through a series of rostopics.
+
+
+## Detecting Faults
```

Comment:
The contents of this README might not be scoped correctly, compared to the ow_faults_detection README, but I'm open to feedback.

Reply:
It looks fine for now.  Can be revised as the package is fleshed out.